artemis/src/main/java/tech/pegasys/artemis/BeaconNode.java

```diff
@@ -101,9 +100,9 @@ public void start() {
       serviceController.startAll();
 
     } catch (final CompletionException e) {
-      STATUS_LOG.log(Level.FATAL, e.toString());
+      LOG.fatal(e.toString());
     } catch (final IllegalArgumentException e) {
-      STATUS_LOG.log(Level.FATAL, e.getMessage());
+      LOG.fatal(e.getMessage());
```

Comment:
Since we're touching this anyway we should fix them to include the full stack trace.  So probably:

```
} catch (fail CompletionException | IllegalArgumentException e) {
    LOG.fatal("Startup failed", e);
}
```

Reply:
Done